Library/Strings.swift

```diff
@@ -4136,17 +4136,17 @@ catch your eye?"
     )
   }
   /**
-   "Unfollow %{friend_name}"
+   "Unfollow %{friend_name}."
 
    - **es**: "Dejar de seguir a %{friend_name}"
    - **de**: "%{friend_name} nicht mehr folgen"
    - **fr**: "Ne plus suivre %{friend_name}"
-   - **en**: "Unfollow %{friend_name}"
+   - **en**: "Unfollow %{friend_name}."
   */
   public static func Unfollow_friend_name(friend_name: String) -> String {
     return localizedString(
       key: "Unfollow_friend_name",
-      defaultValue: "Unfollow %{friend_name}",
+      defaultValue: "Unfollow %{friend_name}.",
```

Comment:
Just another reminder to merge in master so this file doesn't change. This is old.